index.html

```diff
@@ -0,0 +1,733 @@
+        <div class="mobile-nav2">
+            <img class="tnw-red" src="img/tnw.svg" alt="tnw logo">
+        </div>
+        </div>
+        <nav class="mobile-nav">
+           
+            <div class="mobile-img">
+                <img class="tnw-red" src="img/tnw.svg" alt="tnw logo">
+            </div>
+            <div class="mobile-nav1">
+                <div class="mobile-nav-search">
+                    <i class="fas fa-search gray"></i>
+                </div>
+                <div class="mobile-nav-more">
+                    <i class="fas fa-bars orange"></i>
+                </div>
+            </div>
+     
+        </nav>
+    </header>
+    <main>
+        <section>
+            <div class="section1">
+                <img id="tnw-main-article-image" src="img/tnw.svg" alt="The Next Web Logo">
+                <div class="showcase-text">
+                    <h6 class="white showcase-header">INSIGHTS</h6>
+                    <h1 class="white showcase-title">Satoshi Nakamoto: ‘Bitcoin crosses $10K for first time in 2 months’
+                    </h1>
+                    <ul class="gray showcase-social fsmall">
+                        <li>by <a class="gray" href="#">Satoshi Nakamoto</a></li>
+                        <li><i class="far fa-clock"></i> 25 minutes ago</li>
+                        <li><i class="fas fa-share-alt"></i> 6</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="section2">
+                <h4 class="white">India is reportedly looking to ban 275 more Chinese apps — including PUBG</h4>
+                <ul class="gray showcase-social fsmall">
+                    <li>by <a class="gray" href="#">Satoshi Nakamoto</a></li>
+                    <li><i class="far fa-clock"></i> 25 minutes ago</li>
+                    <li><i class="fas fa-share-alt"></i> 6</li>
+                </ul>
+            </div>
+            <div class="section3">
+                <h4 class="white">India is reportedly looking to ban 275 more Chinese apps — including PUBG</h4>
+                <ul class="gray showcase-social fsmall">
+                    <li>by <a class="gray" href="#">Satoshi Nakamoto</a></li>
+                    <li><i class="far fa-clock"></i> 25 minutes ago</li>
+                    <li><i class="fas fa-share-alt"></i> 6</li>
+                </ul>
+            </div>
+        </section>
+        <div class="news-pages">
+        <h2 class="orange news-pages-header">Latest News <i class="fas fa-chevron-right"></i></h2>
+        <section class="latest-news">
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
```

Comment:
- [ ] **All screens** Please add the "See all Details" link
![image](https://user-images.githubusercontent.com/53256139/88812569-7b6bc900-d17d-11ea-8144-9783280e145b.png)